config/routes.rb

```diff
@@ -43,4 +43,17 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools, constraints: StaffConstraint.new do
```

Comment:
Ah, I knew I was missing the authorization check :smile:. I feel like these kind of checks are better in the controller than in the routes.